authentication/models.py

```diff
@@ -1,3 +1,55 @@
 from django.db import models
+from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
+from django.contrib.auth.models import PermissionsMixin
+from validation.validate_password import validate_password_long, validate_password_include_symbols
 
-# Create your models here.
+
+class CustomUserManager(BaseUserManager):
+
+    def create_user(self, person_email, password=None, **extra_fields):
+        person_email = self.normalize_email(person_email)
+        user = self.model(
+            person_email=person_email, **extra_fields)
+        user.set_password(password)
+        user.save()
+        return user
+
+    def create_superuser(self, person_email, password, **extra_fields):
+        user = self.create_user(person_email, password, **extra_fields)
+        user.is_staff = True
+        user.is_superuser = True
+        user.save()
+        return user
+
+
+class CustomUser(AbstractBaseUser, PermissionsMixin):
+    person_email = models.EmailField(max_length=50, unique=True)
+    person_name = models.CharField(max_length=50)
+    person_surname = models.CharField(max_length=50)
+    password = models.CharField(
```

Comment:
Do we store password as is? We should store password hash only.

Reply:
Password field removed from CustomUser model.